README.md

```diff
@@ -21,11 +21,11 @@ Install the package in R by running the following command (requires the devtools
 devtools::install()
 ```
 
-## Run Remote On A Local Machine
```

Comment:
I trust these changes as I don't have a way to test these updated instructions on the remote - but the text sounds correct, thanks for updating it!

Reply:
Vladimir is also testing and updating these docs